drivers/SmartThings/zigbee-lock/src/lock_utils.lua

```diff
@@ -73,4 +74,23 @@ function lock_utils.code_deleted(device, code_slot)
   return lock_codes
 end
 
+function lock_utils.populate_state_from_data(device)
+  if device.data.lockCodes ~= nil and device:get_field(lock_utils.MIGRATION_COMPLETE) ~= true then
+    -- build the lockCodes table
+    local lockCodes = {}
+    for k, v in pairs(device.data.lockCodes) do
+      lockCodes[k] = v
+    end
+    -- Populate the devices `lockCodes` field
+    device:set_field(lock_utils.LOCK_CODES, utils.deep_copy(lockCodes), { persist = true })
+    -- Populate the devices state history cache
+    -- TODO: "main" usage
```

Comment:
I can remove this.  It might make sense to create a `st.constants` or something similar for something like this, but this is pretty well defined so I'm not too concerned about the raw string usage.